service/models/event.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type Event struct {
+	Domain            string       `json:"domain_name"`
+	Delivered    int       `json:"delivered"`
+	Bounced          int       `json:"bounced"`
+}
```

Comment:
Is there a way to make this look cleaner?

Reply:
`go fmt events.go` should be applied I also see it for the `response.go` file. To prevent this from happening in the future we could have a Travis CI integration to check if files are correctly formatted